Automattic/wc-calypso-bridge
# Update remove WC logo

Fixes #79 

Removes the WC logo from the OBW.  Pending https://github.com/Automattic/wc-calypso-bridge/pull/73

### Screenshot
<img width="785" alt="screen shot 2018-10-24 at 3 42 14 pm" src="https://user-images.githubusercontent.com/10561050/47460179-8244fb00-d7a3-11e8-9637-dd33b8f538ca.png">

### Testing

1.  Visit `/wp-admin/admin.php?page=wc-setup`
2.  Check that WooCommerce logo has been removed.
